runtimes/8.1/start-environment-monitoring

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+lastKnownModification=$(date -r /var/www/html/.env "+%s")
+while true; do
+    sleep 1
+
+    lastModified=$(date -r /var/www/html/.env "+%s")
+
+    if [ $lastModified -gt $lastKnownModification ]; then
+        lastKnownModification=$lastModified
+        pid=$(ps -ef | grep -v grep | grep 'server.php' | awk '{print $2}')
+        kill $pid
```

Comment:
This could be simplified a bit. We don't need grep at all because awk does pattern matching very well.
```suggestion
        ps -ef  | awk '/[s]erver.php/ { print $2 }'  | xargs kill
```
The square brackets around the `[s]` is a clever "hack" that matches the process we are looking for but doesn't match itself in the process list. This can also be used with grep.

Or we could use `pgrep` to find the process we want to kill.
```suggestion
        pgrep -fa server.php | cut -d ' ' -f 1 | xargs kill
```